Bindings/Java/Matlab/examples/c3dExport.m

```diff
@@ -0,0 +1,46 @@
+%% Example of using the Matlab-OpenSim class to read in data and do 
+% operations
+
+%% Choose a c3d file
+[filename, path] = uigetfile('*.c3d');
```

Comment:
Are there C3d files available in the Matlab examples dir in the installation? That is, when this GUI pops up, will it show c3d files in that directory?